rumqttd/src/router/routing.rs

```diff
@@ -703,8 +699,13 @@ impl Router {
                             break;
                         }
                     };
+                    self.scheduler.reschedule(id, ScheduleReason::IncomingAck);
+                }
+                Packet::PubComp(pubcomp, _) => {
+                    // Should we just log with info! instead of creating a span?
+                    let span = tracing::info_span!("pubcomp", pkid = pubcomp.pkid);
+                    let _guard = span.enter();
                 }
```

Comment:
I assume current creating this span is redundant because we are not logging any `Event` after we create the span. I also think we should add a `info!` here.